RecoMuon/DetLayers/src/MuonGEMDetLayerGeometryBuilder.cc

```diff
@@ -23,10 +24,26 @@ MuonGEMDetLayerGeometryBuilder::~MuonGEMDetLayerGeometryBuilder() {}
 // Builds etaPartitions (for rechits)
 pair<vector<DetLayer*>, vector<DetLayer*> > MuonGEMDetLayerGeometryBuilder::buildEndcapLayers(const GEMGeometry& geo) {
   vector<DetLayer*> result[2];
-
   for (int endcap = -1; endcap <= 1; endcap += 2) {
     int iendcap = (endcap == 1) ? 0 : 1;  // +1: forward, -1: backward
 
+    // GE0
+    int station0 = 0;
```

Comment:
This loop could be fully included in the next one, provided you start it from `station = GEMDetId::minStationId0`, and apply the few changes suggested below